.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Packages
         run: |
           apk add findutils git python3 py3-pip shellcheck
-          pip3 install tox
+          pip3 install tox==3.4.0
```

Comment:
Why 3.4.0 specifically?   Could you use `tox<4`?

Reply:
I don't remember why anymore :-) Trying tox<4 per your suggestion